README.md

```diff
@@ -4,7 +4,115 @@
+
+```dart
+  /// The main log level map
+  static final Map<Level, fnb.LogLevel?> _logMap = {
+    Level.ALL: fnb.LogLevel.Trace,
+    Level.FINEST: fnb.LogLevel.Trace,
+    Level.FINER: fnb.LogLevel.Trace,
+    Level.FINE: fnb.LogLevel.Debug,
+    Level.CONFIG: fnb.LogLevel.Debug,
+    Level.INFO: fnb.LogLevel.Info,
+    Level.WARNING: fnb.LogLevel.Warn,
+    Level.SEVERE: fnb.LogLevel.Error,
+    Level.SHOUT: fnb.LogLevel.Error,
+    Level.OFF: null,
+  };
+
+  Level _toLogLevel(fnb.LogLevel level) {
+    return _logMap.keys.firstWhere((key) => _logMap[key] == level);
+  }
+```
+
+Then you can use it in ```_logHandler``` callback:
+
+```dart
+  void _logHandler(fnb.LogEntry logEntry) {
+    final logLevel = _toLogLevel(logEntry.level);
+
+    _log.log(logLevel, '${logEntry.tag}: ${logEntry.msg}');
+  }
+```
+
+And finally, you can setup logger in your app using this code:
+
+```dart
+   fnb.setupLogger(
+      level: logLevel,
+      mobileLogger: mobileLogger,
+      logHandler: _logHandler,
+   );
+```
+
+Where ```level``` is your default log level, ```mobileLogger``` is just a flag that you want to use mobile logger (if you want to use your own logger, you can set it to false) and ```logHandler` is a callback that will be called for each log entry.
+
+Please note: you should setup logger before any other calls to bridge.
+
+### Bridge initialization 🎬
+
+To initialize bridge you need to call ```initRustToDartCaller``` function from the package. It will register all callbacks and initialize bridge. You should call it before any other calls to bridge (just call it right after logger setup).
+
+```dart
+  fnb.initRustToDartCaller();
+```
+
+It's a good idea to check how we do it in Sparx wallet in [NekotonRepository](https://github.com/broxus/ever_wallet_flutter_new/blob/main/packages/nekoton_repository/lib/src/nekoton_repository.dart).
+
+### Bridge usage 📈
+
+#### Fixed point numbers 🔝
+
+We used [Fixed](https://pub.dev/packages/fixed) package for all fixed point calculations. So, you should use it too.
+
+Fixed allows you to store and manipulate fixed point numbers. Fixed point numbers are numbers that have a fixed number of digits after the decimal point. You shouldn't use floating point numbers for financial calculations because they are not precise enough. Using strings 🩼 for storing fixed point is also not a good idea because it's not efficient and can produce bugs due to decimal separator difference (, or .) and can't be used in calculations. Fixed point numbers are precise and can be used for financial calculations.
+
+For [example](https://fixed.onepub.dev/constructors):
+
+```dart
+Fixed.fromInt(1234, scale: 3); // == 1.234
+
+Fixed.fromBigInt(BigInt.from(1234), scale: 3); // == 1.234
+
+final t1 = Fixed.fromDecimal(Decimal.fromInt(1), scale: 2); // == 1.00
+
+final t3 = Fixed.parse('1.234'); // == 1.234, scale: 3
```

Comment:
As I know, for `Fixed.parse` `scale` parameter is 0 by default

Reply:
Hmmm.
They said that `If [scale] is not passed it defaults to 2.`, but it actually calculates automatically in the code O_o. Anyway, I'll remove this example from our doc :)

Reply:
removed.